ray-operator/test/support/yaml.go

```diff
@@ -28,32 +28,46 @@ func DeserializeRayClusterYAML(t Test, filename string) *rayv1.RayCluster {
 	t.T().Helper()
 	rayCluster := &rayv1.RayCluster{}
 	err := deserializeYAML(filename, rayCluster)
-	assert.NoError(t.T(), err)
+	require.NoError(t.T(), err)
 	return rayCluster
 }
 
 func DeserializeRayJobYAML(t Test, filename string) *rayv1.RayJob {
 	t.T().Helper()
 	rayJob := &rayv1.RayJob{}
 	err := deserializeYAML(filename, rayJob)
-	assert.NoError(t.T(), err)
+	require.NoError(t.T(), err)
 	return rayJob
 }
 
 func DeserializeRayServiceYAML(t Test, filename string) *rayv1.RayService {
 	t.T().Helper()
 	rayService := &rayv1.RayService{}
 	err := deserializeYAML(filename, rayService)
-	assert.NoError(t.T(), err)
+	require.NoError(t.T(), err)
 	return rayService
 }
 
 func KubectlApplyYAML(t Test, filename string, namespace string) {
 	t.T().Helper()
 	kubectlCmd := exec.CommandContext(t.Ctx(), "kubectl", "apply", "-f", filename, "-n", namespace)
 	err := kubectlCmd.Run()
-	if err != nil {
-		t.T().Fatalf("Failed to apply %s to namespace %s: %v", filename, namespace, err)
-	}
+	require.NoError(t.T(), err)
```

Comment:
I think `err` is not very informative. From what I recall, it seems like just something similar to "exit 1".

Reply:
Updated. Manual error messages have been added to `require.NoError()` calls.